WordPressAuthenticator/Unified Auth/View Related/Site Address/SiteAddressViewController.swift

```diff
@@ -336,8 +355,9 @@ extension SiteAddressViewController {
                 }
 
                 DDLogError(error.localizedDescription)
-                WordPressAuthenticator.track(.loginFailedToGuessXMLRPC, error: error)
-                WordPressAuthenticator.track(.loginFailed, error: error)
+                // TODO: Add new Tracks events
+                // WordPressAuthenticator.track(.loginFailedToGuessXMLRPC, error: error)
+                // WordPressAuthenticator.track(.loginFailed, error: error)
```

Comment:
I understand that it's a TODO, but would it be possible to remove the commented out code, since the TODO already indicates that there is code to be added there at a later time?
Also, I have been thinking of ways to optimize TODOs (especially to avoid forgetting them). One way could be to add a specific keyword so that you can easily search for them (e.g. `TODO - VOICEOVER: ...`), another way could be to add a user defined warning where you have the TODO, but that only applies if you plan to resolve them before submitting a PR (otherwise you would merge additional warnings, which we don't want)

Reply:
> would it be possible to remove the commented out code

FWIW, I'm in favor of leaving the old tracks to facilitate cross-referencing with the new ones. I've been prefacing the old tracks with `Old track:`, if that helps.  (ex: `// TODO: add new tracks. Old track: WordPressAuthenticator.track(.loginPasswordFormViewed)`)

> add a specific keyword

We could do that too, like maybe `// TODO: - Tracks` or something.

Reply:
> I'm in favor of leaving the old tracks to facilitate cross-referencing with the new ones

It's fine, I guess. At the end of the day, what matters is to remove them when the implementation is complete

Reply:
Follow-up after the discussion. These todos should be a consistent format that is easily searchable, so that they can be fixed once the new Tracks implementation is in place. It looks like this:

```
                // TODO: - Tracks.
                // WordPressAuthenticator.track(.loginFailedToGuessXMLRPC, error: error)
                // WordPressAuthenticator.track(.loginFailed, error: error)
```